src/user/mod.rs

```diff
@@ -1,5 +1,19 @@
 mod handler;
-pub mod manager;
 pub mod model;
-pub mod route;
+pub(crate) mod service;
 pub mod util;
+
+use crate::user::handler::{login, logout, me, register};
+use actix_web::web;
+
+pub fn route(cfg: &mut web::ServiceConfig) {
```

Comment:
As we should only have one `route` function I think it is a bit over-killing to create a module specially for it...